### ISSUE TITLE
Change to custom TTF font for NSIS installer instead of Default setting

### ISSUE THREAD

**Issue by F3nn1t0** [nextcloud/client-building#73]
Hey there, I recently tried to compile this installer script into my own Localization which seems to be okay so far, but the default display font is not really great at display the instruction on the installer. So I would like some kind of advice or solution on how can I suggest the NSIS script to use a custom TTF Font instead of the default one.

I have tried using the FontName plugin so far but it does seem to not compatible with the script, maybe because of the MUI2? I am not really sure myself. So is there any workaround or some solution that I can use for this situation?

Note: Not experienced in NSIS script or anything related to compiling Window Software. So any feedback or advice is appreciated, Thanks You.

**Comment by allexzander** [comment]
@F3nn1t0 Hi. FYI We are not going to support the NSIS installer anymore. Please try using MSI instead (**build-installer-msi.bat**).

**Comment by F3nn1t0** [comment]
Fair point, the NSIS seem kinda outdate too. But I have been struggling with the localization for msi installer (I know it is easy to just use English as default but I taking this project for the user that are not English-Fluent as well). Would you guys have any guideline for how can I do the localization of the msi correctly?

**Comment by allexzander** [comment]
@F3nn1t0 I don't have deep experience with that, but, I guess Microsoft Documentation should be a good place to start https://docs.microsoft.com/en-us/windows/win32/msi/localizing-a-windows-installer-package
You can also open a PR here if you've managed to localize it.
